addons/binding/org.openhab.binding.co2signal/src/main/java/org/openhab/binding/co2signal/internal/CO2SignalHandlerFactory.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.co2signal.internal;
+
+import static org.openhab.binding.co2signal.CO2SignalBindingConstants.THING_TYPE_CO2;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.openhab.binding.co2signal.handler.CO2SignalHandler;
+
+/**
+ * The {@link CO2SignalHandlerFactory} is responsible for creating things and thing
+ * handlers.
+ *
+ * @author Jens Viebig - Initial contribution
+ */
+public class CO2SignalHandlerFactory extends BaseThingHandlerFactory {
+
+    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singleton(THING_TYPE_CO2);
```

Comment:
You have already created a SUPPORTED_THING_TYPES_UIDS in your constants class, no need to create a new one here.

Reply:
Ok fixed